src/store/modules/users.spec.js

```diff
@@ -62,9 +55,23 @@ describe('users', () => {
     expect(store.getters['users/byActiveGroup'].map(e => e.id)).toEqual([user1.id, user2.id])
   })
 
-  it('sets routeError if user is not accessible', () => {
+  it('can select user', async () => {
+    expect.assertions(1)
+    mockGet.mockReturnValueOnce(user1)
+    await store.dispatch('users/selectUser', { userId: user1.id })
+    expect(store.getters['users/activeUser'].id).toEqual(user1.id)
+    mockGet.mockReset()
```

Comment:
Why reset them?

If it's needed would be better to reset them all in a before/after hook using [jest.resetAllMocks() ](http://facebook.github.io/jest/docs/en/jest-object.html#jestresetallmocks)

Reply:
It didn't work without resetting them, even if `mockReturnValueOnce` should just mock it a single time. Maybe I'll apply the resetAllMocks thing in future...